homeassistant/components/backup/const.py

```diff
@@ -0,0 +1,20 @@
+"""Constants for the Backup integration."""
+from logging import getLogger
+
+from awesomeversion import AwesomeVersion
+
+from homeassistant.const import __version__ as HAVERSION
+
+DOMAIN = "backup"
+VERSION = AwesomeVersion(HAVERSION)
+LOGGER = getLogger(__package__)
+
+EXCLUDE_FROM_BACKUP = [
```

Comment:
Should we exclude the media folder? (especially now people can upload media)

Reply:
The point of this is to have a way to migrate everything easily from core/container to OS, if your media is missing, it would still require you to;
a) re-upload them in the UI on the new installation
b) install samba, and move them

Both these are not great, so unless we get issues with it, I'd say keep it for mow.